controllers/backstage_controller.go

```diff
@@ -295,14 +301,13 @@ func (r *BackstageReconciler) labels(meta *v1.ObjectMeta, backstage bs.Backstage
 
 // SetupWithManager sets up the controller with the Manager.
 func (r *BackstageReconciler) SetupWithManager(mgr ctrl.Manager, log logr.Logger) error {
-	if len(r.PsqlImage) == 0 {
-		r.PsqlImage = "quay.io/fedora/postgresql-15:latest"
-		log.Info("Enviroment variable is not set, default is used", bs.EnvPostGresImage, r.PsqlImage)
-	}
 
-	if len(r.BackstageImage) == 0 {
-		r.BackstageImage = "quay.io/janus-idp/backstage-showcase:next"
-		log.Info("Enviroment variable is not set, default is used", bs.EnvBackstageImage, r.BackstageImage)
+	const imageNotSet = "environment variable is not set, default will be used:"
+	if _, ok := os.LookupEnv(EnvPostgresImage); !ok {
+		log.Info(imageNotSet, "", EnvPostgresImage)
+	}
+	if _, ok := os.LookupEnv(EnvBackstageImage); !ok {
+		log.Info(imageNotSet, "", EnvBackstageImage)
```

Comment:
```suggestion
		log.Info(imageNotSet, EnvBackstageImage, "")
```

Otherwise, the log message looks weird to me ;)
```
2024-02-09T10:58:24+01:00       INFO    setup   environment variable is not set, default will be used:  {"": "RELATED_IMAGE_backstage"}
```